pkg/apps/blackduck/latest/databaseinit.go

```diff
@@ -110,6 +110,7 @@ func InitDatabase(name string, createHub *blackduckapi.BlackduckSpec, isClusterS
 		"ALTER SYSTEM SET random_page_cost TO '4.0';",
 		"ALTER SYSTEM SET standard_conforming_strings TO 'off';",
 		"ALTER SYSTEM SET temp_buffers TO '16MB';",
+		"ALTER SYSTEM SET shared_buffers TO '1024MB';",
```

Comment:
shared_bufferd must be configured using https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.10.x/pkg/apps/database/postgres/postgres.go#L166